e2e/tests/packet_forward_middleware/forward_timeout_test.go

```diff
@@ -26,459 +27,198 @@ type PFMTimeoutTestSuite struct {
+	// Check that the packet was received on chainB
+	_, err = cosmos.PollForMessage[*chantypes.MsgRecvPacket](ctx, chainB.(*cosmos.CosmosChain), cosmos.DefaultEncoding().InterfaceRegistry, bHeightBeforeTransfer, bHeightBeforeTransfer+20, nil)
 	s.Require().NoError(err)
 
-	bHeight, err = b.Height(ctx)
-	s.Require().NoError(err)
+	time.Sleep(time.Second * 12) // Wait for timeout
 
-	// Poll for the MsgTimeout on chainB and the MsgAck on chainA
-	_, err = cosmos.PollForMessage[*chantypes.MsgTimeout](ctx, b.(*cosmos.CosmosChain), b.Config().EncodingConfig.InterfaceRegistry, bHeight, bHeight+30, nil)
+	// Verify that the users funds are still in escrow on chainA and chainB before we relay the timeout between chainB and chainC
+	userABalance, err := chainA.GetBalance(ctx, userA.FormattedAddress(), chainA.Config().Denom)
 	s.Require().NoError(err)
-
-	_, err = testutil.PollForAck(ctx, a, aHeight, aHeight+30, transferTx.Packet)
+	userBBalance, err := chainB.GetBalance(ctx, userB.FormattedAddress(), ibcTokenB.IBCDenom())
 	s.Require().NoError(err)
 
-	err = testutil.WaitForBlocks(ctx, 1, a)
-	s.Require().NoError(err)
+	s.Require().Equal(testvalues.StartingTokenAmount-transferAmount.Int64(), userABalance.Int64())
+	s.Require().Equal(zeroBal, userBBalance)
 
-	// Assert balances to ensure that the funds are still on the original sending chain
-	chainABalance, err := a.GetBalance(ctx, usrA.FormattedAddress(), a.Config().Denom)
+	escrowBalanceAB, err := chainA.GetBalance(ctx, escrowAddrAB.String(), chainA.Config().Denom)
 	s.Require().NoError(err)
-
-	// Compose the prefixed denoms and ibc denom for asserting balances
-	firstHopDenom := transfertypes.GetPrefixedDenom(baChan.PortID, baChan.ChannelID, a.Config().Denom)
-	secondHopDenom := transfertypes.GetPrefixedDenom(cbChan.PortID, cbChan.ChannelID, firstHopDenom)
-	thirdHopDenom := transfertypes.GetPrefixedDenom(dcChan.PortID, dcChan.ChannelID, secondHopDenom)
-
-	firstHopDenomTrace := transfertypes.ParseDenomTrace(firstHopDenom)
-	secondHopDenomTrace := transfertypes.ParseDenomTrace(secondHopDenom)
-	thirdHopDenomTrace := transfertypes.ParseDenomTrace(thirdHopDenom)
-
-	firstHopIBCDenom := firstHopDenomTrace.IBCDenom()
-	secondHopIBCDenom := secondHopDenomTrace.IBCDenom()
-	thirdHopIBCDenom := thirdHopDenomTrace.IBCDenom()
-
-	chainBBalance, err := b.GetBalance(ctx, usrB.FormattedAddress(), firstHopIBCDenom)
+	escrowBalanceBC, err := chainB.GetBalance(ctx, escrowAddrBC.String(), ibcTokenB.IBCDenom())
 	s.Require().NoError(err)
 
-	chainCBalance, err := c.GetBalance(ctx, usrC.FormattedAddress(), secondHopIBCDenom)
-	s.Require().NoError(err)
+	s.Require().Equal(transferAmount, escrowBalanceAB)
+	s.Require().Equal(transferAmount, escrowBalanceBC)
 
-	chainDBalance, err := d.GetBalance(ctx, usrD.FormattedAddress(), thirdHopIBCDenom)
+	// Relay the packet from chainB to chainC, which should timeout
+	err = relayer.Flush(ctx, s.GetRelayerExecReporter(), s.GetPathByChains(chainB, chainC), chanBC.ChannelID)
```

Comment:
Just so I understand, this function automatically submits the msg timeout back to chain B